packages/twenty-server/src/queue-worker.ts

```diff
@@ -39,7 +39,17 @@ async function bootstrap() {
           .select(JobsModule)
           .get(jobClassName, { strict: true });
 
-        await job.handle(jobData.data);
+        try {
+          await job.handle(jobData.data);
+        } catch (err) {
+          if (
+            !jobData?.opts?.attempts ||
+            jobData.attemptsMade >= jobData.opts.attempts
+          ) {
+            exceptionHandlerService?.captureExceptions([err]);
```

Comment:
I would throw it anyway honestly even at attempt 1, it should not happen

Reply:
Alright 👍 @charlesBochet